content/docs/command-reference/fetch.md

```diff
@@ -22,7 +23,7 @@ of the project, but without placing them in the <abbr>workspace</abbr>. This
 makes the data files available for linking (or copying) into the workspace.
 (Refer to [dvc config cache.type](/doc/command-reference/config#cache).) Along
 with `dvc checkout`, it's performed automatically by `dvc pull` when the target
-[DVC-files](/doc/user-guide/dvc-file-format) are not already in the cache:
+[`.dvc` files](/doc/user-guide/dvc-file-format) are not already in the cache:
```

Comment:
So fetch is very similar to pull. Please try to apply the same changes all over, mentioning both dvc.yaml and .dvc files, etc.